cg/apps/demultiplex/sample_sheet/create.py

```diff
@@ -2,7 +2,10 @@
 from typing import List
 from typing_extensions import Literal
 
-from cg.apps.demultiplex.sample_sheet.novaseq_sample_sheet import SampleSheetCreator
+from cg.apps.demultiplex.sample_sheet.sample_sheet_creator import (
+    SampleSheetCreatorV1,
+    SampleSheetCreatorV2,
+)
```

Comment:
Would we want a naming reflecting NovaSeq vs NovaSeqX? For example SampleSheetCreatorNovaSeq and SampleSheetCreatorNovaSeqX?

Reply:
Good point. The reason I chose this is that HiSeqX uses SampleSheetV1, but we are not doing HiSeqX demultiplexing in cg anyways.

Reply:
We do for old reruns, but they should already have a valid sample sheet in HK

Reply:
I will leave it as V1/V2 for now to make it more friendly in case of the inclusion of other sequencers